hw/ip/alert_handler/data/alert_handler.hjson

```diff
@@ -57,12 +57,14 @@
       default: "1",
       local: "true"
     },
-    { name: "AlertLpgMap",
+    { name: "LpgMap",
       desc: '''
             Defines a mapping from alerts to LPGs.
             '''
       type: "logic [NAlerts-1:0][NLpgWidth-1:0]",
-      default: "'0",
+      default: '''
+'0
```

Comment:
nit, spacing seems a little weird here.

Reply:
I've aligned the template to print this out more nicely